feat(em): improved bandwidth option (#3299)
scaleway/docs-content: bare-metal/elastic-metal/how-to/adjust-available-bandwidth.mdx

---
meta:
  title: How to adjust the available bandwidth for an Elastic Metal server
  description: Discover step-by-step instructions for adjusting (upgrading or downgrading) the available bandwidth for your Elastic Metal server.
content:
  h1:  How to adjust the available bandwidth for an Elastic Metal server
  paragraph: Discover step-by-step instructions for adjusting (upgrading or downgrading) the available bandwidth for your Elastic Metal server.

> tags: dedibox connect putty

Reviewer: ```suggestion
tags: elastic-metal connect putty
```

dates:
  validation: 2024-06-11
  posted: 2024-06-11
---

> Every Elastic Metal dedicated server comes with a designated bandwidth allowance. Increase your server's connectivity with additional, high-performance bandwidth options of up to 25Gbps, all with unlimited traffic.

Reviewer 2: ```suggestion
Every Elastic Metal server comes with a designated bandwidth allowance. Increase your server's connectivity with additional, high-performance bandwidth options of up to 25Gbps, all with unlimited traffic.
```

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- An Elastic Metal server [installed](/bare-metal/elastic-metal/quickstart/#how-to-create-an-elastic-metal-server)

## Upgrading bandwidth for your Elastic Metal server

Follow these steps to boost the bandwidth of your Elastic Metal server:

1. Navigate to **Elastic Metal** under the **Bare Metal** section in the [Scaleway console](https://console.scaleway.com) side menu. This action takes you to the Elastic Metal dashboard.
2. Click the server name or the <Icon name="more" /> beside your desired server to access its information page.

> 3. Select **Settings** to reveal a list of available configurations. Then, proceed to the **Public bandwidth** section and click **Manage bandwidth**. A pop-up window will appear.

Reviewer 2: ```suggestion
3. Select **Settings** to reveal a list of available configurations. Then, proceed to the **Public bandwidth** section and click **Manage bandwidth**. A pop-up window appears.
```

4. Choose the desired maximum bandwidth for your Elastic Metal server. The associated monthly fee displays on the right side of the pop-up.
    <Message type="note">
      The maximum available bandwidth for each Elastic Metal server is determined by the physical conditions of the network card installed, as specified in the offer description.
    </Message>
5. Click **Continue** to confirm your selected bandwidth. A summary of your order displays

> 6. Finally, click **Update public bandwidth** to initiate the bandwidth upgrade process for your Elastic Metal server.

Reviewer 3: ```suggestion
6. Click **Update public bandwidth** to initiate the bandwidth upgrade process for your Elastic Metal server.
```

    <Message type="tip">
      The initial month's fee for your chosen bandwidth option will be billed prorated.
    </Message>

## Downgrading bandwidth for your Elastic Metal server

To reduce the bandwidth for your Elastic Metal server, follow these steps:

1. Access the **Elastic Metal** section within the **Bare Metal** category of the [Scaleway console](https://console.scaleway.com).
2. Click the server name or the <Icon name="more" /> beside your desired server to access its information page.
3. Navigate to **Settings** and then to the **Public bandwidth** section. Click **Manage bandwidth** to open a pop-up.

> 4. Adjust the bandwidth to your desired level. The associated monthly fee will adjust accordingly.

Reviewer: ```suggestion
4. Adjust the bandwidth to your desired level. The associated monthly fee will be adjusted accordingly.
```

    <Message type="tip">
      Sliding the bar to the far left will reduce the bandwidth to the value initially included with your Elastic Metal server.
    </Message>

> 5. After confirming your selection by clicking **Continue**, review the order summary, and then click **Update public bandwidth** to execute the bandwidth downgrade process.

Reviewer 4: ```suggestion
5. Click **Continue** to confirm your selection after reviewing the order summary.
6. Click **Update public bandwidth** to execute the bandwidth downgrade process.
```
